client/components/OperationList.js

```diff
@@ -530,4 +530,3 @@ export default class OperationsComponent extends React.Component {
         );
     }
 }
-
```

Comment:
I do not understand why the empty last line is removed whereas I still have it on my local copy…